client/templates.ts

```diff
@@ -6,11 +6,22 @@ createTemplateForComponent('MessageActions', () => import('./components/Message/
 
 createTemplateForComponent('reactAttachments', () => import('./components/Message/Attachments'));
 
-createTemplateForComponent('ThreadMetric', () => import('./components/Message/Metrics/Thread'));
+createTemplateForComponent('ThreadMetric', () => import('./components/Message/Metrics/Thread'), {
+	renderContainerView: () =>
+		HTML.DIV({
+			style: 'min-height: 36px;',
+		}),
+});
 
 createTemplateForComponent(
 	'DiscussionMetric',
 	() => import('./components/Message/Metrics/Discussion'),
+	{
```

Comment:
Code style consistence... braces in the same line or not?

Reply:
tell that to my prettier

Reply:
Yep, under `client/`, Prettier is the ruler.